gremlin-driver/src/test/java/org/apache/tinkerpop/gremlin/driver/WebSocketClientBehaviorIntegrateTest.java

```diff
@@ -271,4 +277,14 @@ public void shouldNotCreateReplacementConnectionWhenClientClosesConnection() thr
                         .filter(str -> str.contains("Considering new connection on"))
                         .count());
     }
+
+    @Test
+    public void shouldUseEpollIfAvailable() throws Exception {
+        final WebSocketClient client = new WebSocketClient();
+        if (Epoll.isAvailable()) {
+            assertTrue(client.group instanceof EpollEventLoopGroup);
```

Comment:
Nit: `assertTrue(client.group instanceof (Epoll.isAvailable() ? EpollEventLoopGroup : NioEventLoopGroup));`

Reply:
This gives an `Expression expected` error on the two values, which I guess it's because the ternary operator is looking for a variable to assign the values to based on the condition but we are using this as a statement.